lib/vhost/vhost_user.c

```diff
@@ -342,6 +344,35 @@ vhost_user_get_features(struct virtio_net **pdev,
 	return RTE_VHOST_MSG_RESULT_REPLY;
 }
 
+static int
+vhost_user_get_config(struct virtio_net **pdev,
+			struct vhu_msg_context *ctx,
+			int main_fd __rte_unused)
+{
+	struct virtio_net *dev = *pdev;
+	struct rte_vdpa_device *vdpa_dev;
+
+	if (validate_msg_fds(dev, ctx, 0) != 0)
+		return RTE_VHOST_MSG_RESULT_ERR;
+
+	vdpa_dev = dev->vdpa_dev;
+
+	if (vdpa_dev)
+		vdpa_dev->ops->get_dev_config(dev->vid,ctx->msg.payload.cfg.region,ctx->msg.payload.cfg.size);
```

Comment:
white space after, ","

split to two lines.

Please run kernel checkpatch.pl before pushing changes to catch such silly cosmetic changes.